docker/docker_republish.sh

```diff
@@ -22,28 +22,32 @@ function usage {
 
 DOCKERHUB_TAG=;
 OUTPUT_TAG=;
-TARGET_REPO=docker.io
-TARGET_ORG=diem
-DISABLE_TRUST=false
+TARGET_REPO="docker.io"
+TARGET_ORG="diem"
+DISABLE_TRUST="false"
+IMAGES="init faucet tools validator validator_tcb cluster_test client"
```

Comment:
nice array here to remove lots of redundant code with a loop

Reply:
yeah I'm torn, the simpler script was more readable/recoverable if a human had to step in for any reason, now you'd need to reason about the script.  shrug.

Reply:
yea, but most case people touch this part from our team 😆